packages/core/src/odata-common/request/odata-request-config.ts

```diff
@@ -25,6 +25,7 @@ export abstract class ODataRequestConfig {
   private _customHeaders: Record<string, string> = {};
   private _customQueryParameters: Record<string, string> = {};
   private _customRequestConfiguration: Record<string, string> = {};
+  private _appendPaths: string[] = [];
```

Comment:
The naming here sounds like a boolean  i.e. "should I append paths?"
Consider `_appendedPaths`, `_customPaths` or something in this direction.